logger.js

```diff
@@ -64,8 +64,13 @@ function pinoLogger (opts, stream) {
   const autoLoggingIgnore = opts.autoLogging && opts.autoLogging.ignore ? opts.autoLogging.ignore : null
   delete opts.autoLogging
 
+  const onRequestReceivedObject = opts.customReceivedObject && typeof opts.customReceivedObject === 'function' ? opts.customReceivedObject : undefined
```

Comment:
Could you move the function to be a top-level function instead of a closure? I think the code would read better.